src/components/Progress/Progress.js

```diff
@@ -1,14 +1,16 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
 import propTypes from 'prop-types';
 
-import styled from 'styled-components';
+import styled, { keyframes, css } from 'styled-components';
 
 import Cutout from '../Cutout/Cutout';
 import { blockSizes } from '../common/system';
 
 const Wrapper = styled(Cutout)`
```

Comment:
I think `Wrapper` sounds too vague when reading the render of `Progress`, what do you think about renaming it to something like `CutoutWrapper`?

Reply:
I'll leave it like this since we already use similar approach in couple of other components